gammapy/background/tests/test_energy_offset_array.py

```diff
@@ -0,0 +1,24 @@
+from ..energy_offset_array import EnergyOffsetArray
+from gammapy.data import DataStore
+from gammapy.data import EventList
```

Comment:
We're using relative explicit imports consistently within Gammapy, i.e. here replace:
```python
from gammapy.data import DataStore
from gammapy.data import EventList
```
with
```python
from ...data import DataStore, EventList
```